app/domain/logs.rb

```diff
@@ -290,19 +290,19 @@ module AuthnJwt
         code: "CONJ00056D"
       )
 
-      JWTAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
+      JwtAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
         msg: "Started JWT authentication flow for vendor '{0}'",
         code: "CONJ00057D"
       )
 
-      URL_IDENTITY_PROVIDER_SELECTED = ::Util::TrackableLogMessageClass.new(
-        msg: "The JWT ID is included in and will be retrieved from the URL",
+      SelectingIdentityProviderInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Selecting identity provider interface...",
         code: "CONJ00058D"
       )
 
-      DECODED_TOKEN_IDENTITY_PROVIDER_SELECTED = ::Util::TrackableLogMessageClass.new(
-        msg: "JWT Identity in decoded token is available and will be retrieved from it",
-        code: "CONJ00059D"
+      SelectedIdentityProviderInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Selected identity provider interface is: '{0-identity-provider-interface-name}'",
```

Comment:
fine

Reply:
maybe we don;t need "is"
```suggestion
        msg: "Selected identity provider interface: '{0-identity-provider-interface-name}'",
```